styles/globals.css

```
@@ -641,6 +641,41 @@ video {
   animation: appear 0.3s ease-in reverse forwards;
 }
 
+@keyframes dropdownIn {
```

Stylelint (17.14.1)

[error] Expected keyframe name "dropdownIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)